api/openapi/model-registry.yaml

```
@@ -0,0 +1,1041 @@
+    ModelArtifactListResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ModelArtifactList'
+      description: 'A response containing a list of ModelArtifact entities. '
+    ModelArtifactResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ModelArtifact'
+      description: 'A response containing a `ModelArtifact` entity. '
+    ModelVersionListResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ModelVersionList'
+      description: 'A response containing a list of `ModelVersion` entities. '
+    ModelVersionResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ModelVersion'
+      description: 'A response containing a `ModelVersion` entity. '
+    RegisteredModelListResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/RegisteredModelList'
+      description: 'A response containing a list of `RegisteredModel` entities. '
+    RegisteredModelResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/RegisteredModel'
+      description: 'A response containing a `RegisteredModel` entity. '
+    ArtifactResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Artifact'
+      description: 'A response containing an `Artifact` entity. '
+    ArtifactListResponse:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ArtifactList'
+      description: 'A response containing a list of `Artifact` entities. '
+  parameters:
+    id:
+      name: id
+      description: The ID of resource.
+      schema:
+        type: string
+      in: path
+      required: true
+    name:
+      examples:
+        name:
+          value: artifact-name
+      name: name
+      description: Name of entity to search.
+      schema:
+        type: string
+      in: query
+      required: false
+    externalID:
+      examples:
+        externalID:
+          value: '10'
+      name: externalID
+      description: External ID of entity to search.
+      schema:
+        type: string
+      in: query
+      required: false
+    pageSize:
+      examples:
+        pageSize:
+          value: '100'
+      name: pageSize
+      description: Number of entities in each page.
+      schema:
+        type: string
+      in: query
+      required: false
+    nextPageToken:
+      examples:
+        nextPageToken:
+          value: IkhlbGxvLCB3b3JsZC4i
+      name: nextPageToken
+      description: Token to use to retrieve next page of results.
+      schema:
+        type: string
+      in: query
+      required: false
+    orderBy:
+      style: form
+      explode: true
+      examples:
+        orderBy:
+          value: name asc
```

```suggestion
          value: "CREATE_TIME"
```

We didn't get also if ` asc` or ` desc` postfix is supported in enum? 🤔

---

Yup, meant to add another query parameter for order direction.